File: petManage_api/src/main/java/dot/petstore/model/Pet.java
package dot.petstore.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Pet {
    private int idPet;
    private int idCliente;
    private String nome;
    private String raca;
    private String sexo;
    private String dtNascimento;
    private double peso;
}

File: petManage_app/src/app/model/pet.ts
export class Pet {
    idPet: number;
    idCliente: number;
    nome: string;
    raca: string;
    sexo: string;
    dtNascimento: string;
    peso: number;

    constructor(){
        this.idPet = 0;
        this.idCliente = 0;
        this.nome = "";
        this.sexo = "";
        this.raca = "";
        this.peso = 0;
        this.dtNascimento = new Date().toISOString();
    }
}